src/main/groovy/org/labkey/gradle/task/RunUiTest.groovy

```diff
@@ -88,11 +88,9 @@ abstract class RunUiTest extends Test
         systemProperty "labkey.root", project.rootProject.projectDir
         systemProperty "project.root", project.rootProject.projectDir
         systemProperty "user.home", System.getProperty('user.home')
-        // A handfull of tests require tomcat.home to be defined when running within IntelliJ
+        // A handful of tests require tomcat.home to be defined when running within IntelliJ
         systemProperty "tomcat.home", project.tomcat.catalinaHome
```

Comment:
In my full enlistment, there is only one reference to `tomcat.home`, in a method that is only called twice (`TestProperties.getTomcatHome`). Your testAutomation PR removes one and should remove the other as well (I left feedback in that PR)
```suggestion
```